src/test/java/de/doubleslash/usb_led_matrix/SettingsTest.java

```diff
@@ -109,4 +115,21 @@ void shouldSetComPortAndNumberOfLedsWhenConfiguredWithShortCommandLineOption() {
       assertThat(Settings.getNumberOfLeds()).isEqualTo(5);
       assertThat(Settings.getCom()).isEqualTo("COM7");
    }
+
+   @Test
+   public void testGetColor() {
+      assertEquals(Color.GREEN, AvailabilityStatus.Available.getColor());
+      assertEquals(Color.ORANGE, AvailabilityStatus.Away.getColor());
+      assertEquals(Color.ORANGE, AvailabilityStatus.BeRightBack.getColor());
+      assertEquals(Color.RED, AvailabilityStatus.Busy.getColor());
+      assertEquals(Color.RED, AvailabilityStatus.DoNotDisturb.getColor());
+      assertEquals(Color.BLACK, AvailabilityStatus.Offline.getColor());
+   }
+
+   @ParameterizedTest
+   @EnumSource(AvailabilityStatus.class)
+   public void setColorTest(AvailabilityStatus status) {
+      status.setColor(Color.PINK);
+      assertEquals(Color.PINK, status.getColor());
+   }
```

Comment:
These tests are very close to implementation.
Please write tests that reflect the actual use cases, e.g.:
- test if the colors are configured correctly when being set in the UI
- test if the colors are persisted in the properties file when configured and saved in the UI
- test if the colors are loaded correctly from the properties file
- test what happens in case there is no properties file

And please use the usual naming convention for tests (should.......When........).

Reply:
Done.